.github/workflows/ansible-lint.yml

```
@@ -0,0 +1,20 @@
+name: Ansible Lint # feel free to pick your own name
+on: [push, pull_request]
```

Do we need to run the CI for non main branches?
```suggestion
on:
  push:
    branches: [main]
  pull_request:
    branches: [main]
```

---

Not sure if that works, as I do not create a pull_request from `main` to `main`.

---

I always just take what the action's authors provide:
https://github.com/ansible/ansible-lint-action

---

The parameter is not about the branch you're coming from but to the one you're merging to. The current state  means that the CI will run every time you push to any branch and create a PR to merge to any branch. I guess that since it's a simple run that doesn't consume our resources, we can keep it running on all these cases